dashboard/config/puma.rb

```diff
@@ -26,7 +26,18 @@
   Cdo::AppServerHooks.before_fork
 end
 
+# Statsig is initialized here for managed environments. For development, it is
+# intialized in config/initializers/statsig.rb
+require 'statsig'
+# Determine if this is a managed test environment
+managed_test_environment = CDO.running_web_application? && CDO.test_system?
+# Enable local_mode for all environments except prod and test. This limits the
+# amount of metrics we emit, thereby lowering our credit usage.
+local_mode = CDO.rack_env?(:production) || managed_test_environment ? false : true
+options = StatsigOptions.new({'tier' => CDO.rack_env}, network_timeout: 5, local_mode: local_mode)
+
 on_worker_boot do |_index|
+  Statsig.initialize(CDO.statsig_server_secret_key, options)
```

Comment:
Do we need to initialize `Statsig` here if we also doing it within `Cdo::AppServerHooks`?

Reply:
Ah you're right, I'm just now seeing the Cdo::AppserverHooks.after_fork get's called in puma. I'll go ahead and remove this explicit initialization in puma, and keep it in AppserverHooks.